.circleci/config.yml

```diff
@@ -9,6 +9,32 @@ executors:
       - image: cimg/go:1.14
 
 commands:
+  verify_dist_files_exist:
+    parameters:
+      files:
+        type: string
+        default: |
+            bin/otelcol_darwin_amd64
+            bin/otelcol_linux_arm64
+            bin/otelcol_linux_amd64
+            bin/otelcol_windows_amd64.exe
+            dist/otel-collector-*.arm64.rpm
+            dist/otel-collector_*_amd64.deb
+            dist/otel-collector-*.x86_64.rpm
+            dist/otel-collector_*_arm64.deb
```

Comment:
Also `dist/otel-collector.msi`?

Reply:
`ls dist` and `ls bin` did not reveal it but probably I had disabled msi build job accidentally while testing. Added now.